test.mk

```diff
@@ -23,4 +23,4 @@ test-functional: test-kubernetes
 
 test-kubernetes: ginkgoBuild
 	make -C ./test/e2e build
-	@ORCHESTRATOR=kubernetes bash -c 'pgrep ssh-agent || eval `ssh-agent` && ./test/e2e/bin/e2e-runner'
+	@ORCHESTRATOR=kubernetes bash -c 'eval `ssh-agent` && ./test/e2e/bin/e2e-runner'
```

Comment:
eval `ssh-agent` must be run. Otherwise there will be an error like "SSH_AUTH_SOCK not setup" during the test suite. This was true in both local development and pipeline run.

Reply:
why is now throwing an error?

Reply:
resolved, no longer removing `pgrep ssh-agent`